frontend/assets/style/_layout.scss

```diff
@@ -1,8 +1,10 @@
+$bannerGeneralHeight: 2.3rem;
+
 .l-no-navigation {
   display: flex;
   justify-content: center;
   align-items: flex-start;
-  height: 100%;
+  height: 100vh;
```

Comment:
Probably good idea to check that all the height 100vh work well on mobile because the viewport is jumpy with the search bar. (in some case where there is a lot of space around, it's ok) Same comment apply everywhere where you calcul the space with the vertical height available. If we can't test now, we should add a reminder to double check all those properties

Reply:
Yes, 100vh is a tricky subject, we'll need to review it after we can successfully load GI on a real mobile phone. This won't be forgot for sure.

Reply:
So, for a reminder, will a comment do? Or is it worth opening an issue as well?

Reply:
In our minds it's enough. This will be the first we'll notice when we got GI running on a mobile device